index.js

```diff
@@ -1,29 +1,49 @@
-function getTopLevelBlocksWithoutDirAttribute() {
-  return document.querySelectorAll('.notion-page-content > div[data-block-id]:not([dir])')
+
+function alignListItemsToRight() {
+  const items = getListItems();
+
+  items.forEach((item) => {
+    if (isRTL(item.innerText)) {
+      item.style['text-align'] = 'right';
+    }
+  });
+}
```

Comment:
Hello @dotanrs 

Thanks for your contribution and consideration here.
What do you think of using `text-align: start;` instead?

This should allow us to avoid checking if it's RTL/LTR. This is in line with the approach that we are already using with direction (auto, let the browser handle it).

Reply:
I love it. Changed it